src/main/java/meteordevelopment/meteorclient/mixin/BackgroundRendererMixin.java

```diff
@@ -27,4 +29,9 @@ private static void onApplyFog(Camera camera, BackgroundRenderer.FogType fogType
             }
         }
     }
+
+    @Inject(method = {"getFogModifier(Lnet/minecraft/entity/Entity;F)Lnet/minecraft/client/render/BackgroundRenderer$StatusEffectFogModifier;"}, at = @At("HEAD"), cancellable = true)
+    private static void onGetFogModifier(Entity entity, float tickDelta, CallbackInfoReturnable<Object> info) {
+        if (Modules.get().get(NoRender.class).noBlindness()) info.setReturnValue(null);
```

Comment:
Why are there two settings if they both do literally the same?

Reply:
It only removes blindness fog

Reply:
Oh, accidentally overlooked these other things and commented on the wrong thing lol, sorry for the wrong review
Doing literally anything on phones is just so terrible :>(